Modules/Core/Transform/include/itkKernelTransform.hxx

```diff
@@ -508,6 +508,43 @@ KernelTransform<TParametersValueType, VDimension>::GetFixedParameters() const ->
 }
 
 
+template <typename TParametersValueType, unsigned int VDimension>
+bool
+KernelTransform<TParametersValueType, VDimension>::GetInverse(Self * inverseTransform) const
+{
+  if (!inverseTransform)
+  {
+    return false;
+  }
+
+  auto deepCopyLandmarks = [](const typename PointSetType::ConstPointer source,
+                              typename PointSetType::Pointer            destination) {
+    typename PointSetType::PointsContainer::ConstIterator sourceIt = source->GetPoints()->Begin();
+
+    typename PointSetType::PointIdentifier i{ 0 };
+    while (sourceIt != source->GetPoints()->End())
+    {
+      destination->SetPoint(i, sourceIt->Value());
+      ++i;
+      ++sourceIt;
+    }
+  };
+
+  // make a deep copy of the source and target landmarks
+  typename PointSetType::Pointer sourceLandmarks = PointSetType::New();
+  typename PointSetType::Pointer targetLandmarks = PointSetType::New();
```

Comment:
Please consider using `auto` here. For the return type of `Type::New()` we have agreed that `auto` is appropriate: https://github.com/InsightSoftwareConsortium/ITKSoftwareGuide/blob/d1562dd6dbc929ba18fdf8ce8aa86e8337597d7e/SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex#L1558-L1562

Reply:
`const auto` would be even nicer  😇 

```cpp
    const auto sourceLandmarks = PointSetType::New();
    const auto targetLandmarks = PointSetType::New();
```